exercises/practice/word-count/word_count.go

```diff
@@ -1 +1,7 @@
 package wordcount
+
+type Frequency map[string]int
+
+func WordCount(phrase string) Frequency {
```

Comment:
I tried to replace the `Frequency` type with `map[string]int` but the tests failed locally.

```sh
go test
# wordcount [wordcount.test]
./cases_test.go:10:14: undefined: Frequency
./cases_test.go:15:3: undefined: Frequency
./cases_test.go:20:3: undefined: Frequency
./cases_test.go:25:3: undefined: Frequency
./cases_test.go:30:3: undefined: Frequency
./cases_test.go:35:3: undefined: Frequency
./cases_test.go:40:3: undefined: Frequency
./cases_test.go:45:3: undefined: Frequency
./cases_test.go:50:3: undefined: Frequency
./cases_test.go:55:3: undefined: Frequency
./cases_test.go:55:3: too many errors
FAIL	wordcount [build failed]
```

Commands to reproduce:

```sh
$ cd exercises/practice/word-count
$ cat .meta/example.go \
| sed "s/type Frequency.*//g" \
| sed "s/Frequency/map\[string\]int/g" \
> word_count.go
```

What do we think? Should we leave this as is? Should we remove the references to `Frequency` in `cases_test.go`?

Reply:
I think we should leave it as-is. In the API comment on the test file, the type was given to the student, so it's ok to give away the type here too.

What are you trying to test by replacing the `Frequency` type with `map[string]int`?  For your example to work we would have to do that replacement also in `cases_test.go`, which in turn would require to adjust the `gen.go` test generator for the exercise, but I don't see why that replacement would be useful in the first place.

Reply:
I agree to keep `type Frequency map[string]int` in there as is.

Reply:
Gotcha. Thanks for clarifying!